app/src/interfaces/divider/divider.vue

```diff
@@ -7,6 +7,7 @@
 			'--v-divider-label-color': color,
 		}"
 		large
+		:inlineTitle="inlineTitle"
```

Comment:
Use html-attr style:

```suggestion
		:inline-title="inlineTitle"
```